### PR TITLE
cassandra 3.11.7

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by Rylan12 — COMMENTED** [review]
Do resources need to be bumped? I don't see an obvious `requirements.txt` to know if the versions are pinned.

File: Formula/cassandra.rb [hunk 1]
```diff
@@ -19,23 +18,24 @@ class Cassandra < Formula
...
+    url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
+    sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
   end
 
+  # thrift==0.9.3
   resource "thrift" do
-    url "https://files.pythonhosted.org/packages/c6/b4/510617906f8e0c5660e7d96fbc5585113f83ad547a3989b80297ac72a74c/thrift-0.11.0.tar.gz"
-    sha256 "7d59ac4fdcb2c58037ebd4a9da5f9a49e3e034bf75b3f26d9fe48ba3d8806e6b"
+    url "https://files.pythonhosted.org/packages/ae/58/35e3f0cd290039ff862c2c9d8ae8a76896665d70343d833bdc2f748b8e55/thrift-0.9.3.tar.gz"
+    sha256 "dfbc3d3bd19d396718dab05abaf46d93ae8005e2df798ef02e32793cd963877e"
```
Comment (chenrui333) [inline Formula/cassandra.rb:38]: Downgraded this dep per requirements.txt

**Review by Rylan12 — DISMISSED** [review]

**Comment by Rylan12** [comment]
Setuptools won't work for python 2 starting at version 45. I downgraded back to version 44.1.1 and added a comment sating it shouldn't be updated in the future

**Review by Rylan12 — APPROVED** [review]

**Comment by chenrui333** [comment]
> Setuptools won't work for python 2 starting at version 45. I downgraded back to version 44.1.1 and added a comment sating it shouldn't be updated in the future

👍 💯

**Pull request closed without merging**